ui/src/components/core/input/input-file.styles.ts

```diff
@@ -16,7 +16,7 @@ export abstract class InputFileStyles {
     transition: 'border-color 0.2s ease-in-out',
     borderWidth: '$default',
     borderRadius: '$lg',
-    zIndex: '$docked',
+    zIndex: '$base',

```

Comment:
Does this element needs z-index? It seems to be static, z-index would be not applied.